secure_uninstall/__openerp__.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
```

Comment:
You can generate a new pr to MT to change our current rule:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms